app/scripts/analysis/tools/powercalc/PowercalcAlgorithms.service.coffee

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+BaseModuleDataService = require 'scripts/BaseClasses/BaseModuleDataService.coffee'
+
+module.exports = class PowercalcAlgorithms extends BaseModuleDataService
+  @inject 'app_analysis_powercalc_msgService'
+
+  initialize: ->
+    @msgManager = @app_analysis_powercalc_msgService
+
+
+  ############
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+  get_cfap_help: ()->
```

Comment:
class method naming convention is to follow camelCase without underscores, i.e. it should be `getCfapHelp`